### ISSUE TITLE
NuGet UI in Visual Studio should not offer to consolidate Microsoft.NETCore.App

### ISSUE THREAD

**Issue by bradwilson** [NuGet/Home#6081]
## Details about Problem

NuGet product used: Visual Studio UI

NuGet version (x.x.x.xxx): 4.4.0

VS version (if appropriate): 2017 15.4.1

OS version (i.e. win10 v1607 (14393.321)): Win10 v1709 (16299.19)

Worked before? If so, with which NuGet version: Unknown

## Detailed repro steps so we can see the same problem

1. Create a solution with projects using various levels of .NET Core. The screenshot provided here uses the `console` project here: https://github.com/xunit/xunit.integration/tree/master/console

2. Right click on the solution, choose 'Manage NuGet Packages for Solution...'

3. The 'Consolidate' tab offers to consolidate different versions of .NET Core (aka, `Microsoft.NETCore.app`).

![VS 2017 15.4.1](https://user-images.githubusercontent.com/16944/31868874-8dcf7c1e-b758-11e7-9c3c-84b57298db14.PNG)

Simply put: **The UI should not offer to do that.**

The correct behavior appears to already exist for the 'Updates' tab (i.e., that `Microsoft.NETCore.App` is not something offered to upgrade), so it's confusing why this logic wasn't also extended to the 'Consolidate' tab.

**Comment by rrelyea** [comment]
@bradwilson  - from the screenshot it looks like one is not allowed to change the versions, but they are allowed to see the different versions installed. Is that correct?

**Comment by bradwilson** [comment]
That's right. The Update tab does not offer updates (which is the correct
behavior).

**Comment by uygary** [comment]
Correct me if I'm wrong @bradwilson but it seems that the Consolidate view doesn't allow updates as well.
In that case, this is by all means already the correct behavior. Consolidate tab should _display_ the version conflict, but it should not allow an _update_ to be performed.
I believe this issue should be closed.
